src/main/java/frc/robot/subsystems/HangSubsystem.java

```diff
@@ -52,16 +68,28 @@ public void liftToHeight() {
     }
   }
 
+  public double liftPosition() {
+    return liftEncoder.getPosition();
+  }
+
   public void winchAndLowerLift() {
     // liftMotor.set(-0.42);
     // winchMotor.set(1);
-    if (Math.abs(liftEncoder.getPosition()) > 1) {
-      liftMotor.set(-HangConstants.LIFT_SPEED);
-      winchMotor.set(1);
-    } else {
-      liftMotor.stopMotor();
-      winchMotor.stopMotor();
-    }
+    // if (Math.abs(liftEncoder.getPosition()) > 1) {
+    liftMotor.set(-HangConstants.LIFT_SPEED);
+    // winchMotor.set(1);
+    // } else {
+    // liftMotor.stopMotor();
+    // winchMotor.stopMotor();
+    // }
```

Comment:
For all this code here we'd likely just want to setup a `CommandGroup` and map that in `RobotContainer` to whatever this was assigned.

```java
new ParallelCommandGroup(
  new LowerLift(liftSubsystem),
  new StartWinch(liftSubsystem)
);
```

`LowerLift` already uses a constant to stop, but we probably want something other than `winchStart()` for the command since we've verified the system works, maybe: `WinchOffMat()` or something better named that runs the winch for some number of encoder rotations.

WDYT? 🤔 
https://docs.wpilib.org/en/latest/docs/software/commandbased/command-groups.html